### PR TITLE
Update to latest arclight

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -11,9 +11,13 @@
 
   devise_for :users
   concern :exportable, Blacklight::Routes::Exportable.new
+        
+    concern :hierarchy, Arclight::Routes::Hierarchy.new
```
Comment (corylown) [inline config/routes.rb:15]: Same as the next line
```suggestion
```

File: config/routes.rb [hunk 2]
```diff
@@ -11,9 +11,13 @@
...
   concern :hierarchy, Arclight::Routes::Hierarchy.new
 
   resources :solr_documents, only: [:show], path: '/catalog', controller: 'catalog' do
+          
+    concerns :hierarchy
```
Comment (corylown) [inline config/routes.rb:20]: Same as the next line
```suggestion
```

**Review by corylown — CHANGES_REQUESTED** [review]
This adds a couple of duplicate lines in routes.rb

File: config/routes.rb (hunk 1 above)
Comment (jcoyne) [reply config/routes.rb:15]: We might want to record this as a flaw in the arclight:update generator.

**Review by corylown — APPROVED** [review]

**Pull request merged**